plugins: 
  - {name: static,  type: "costmap_2d::StaticLayer"} 
  - {name: inflation,  type: "costmap_2d::InflationLayer"} 
publish_frequency: 10.0
global_frame: "odom"
robot_base_frame: "base_link"
footprint: "[ [0.5,0.25],[0.5,-0.25],[-0.5,-0.25],[-0.5,0.25] ]"
map_topic: "/projected_map"
inflation: {
  inflation_radius: 0.2,
  cost_scaling_factor: 10,
}